peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx

```diff
@@ -120,14 +122,13 @@ const CollaborationServiceIntegratedView: React.FC = () => {
 
   const handleLangChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
     setLanguage(parseInt(e.target.value));
-    setSyntaxFullLang(e.target.textContent!);
-    setSyntaxLang(e.target.value === '63' ? 'javascript' 
```

Comment:
Are these changes necessary? We would preferably want to minimize unnecessary formatting changes - they affect both code contributions and unduely change the formatting that we are already familiar with, let alone accidental changes that might affect code functionality (such as the above comment).

Reply:
this line is unchanged, unsure why it shows as removed